components/ChartData.tsx

```diff
@@ -70,7 +70,7 @@ const ChartData = () => {
               <Typography variant="h5" className={classes.value}>
                 $ {numbro(data.chain.tokenPrice.usd).format("0.00")}
               </Typography>
-              : <NotAvailable />}
+              : <NotAvailable variant="h5" className={classes.value} />}
```

Comment:
Using `<h5>` is too large for mobile devices

![image](https://user-images.githubusercontent.com/932636/88905546-f8379980-d288-11ea-8a95-078c94da94ee.png)